errortypes/code.go

```diff
@@ -0,0 +1,29 @@
+package errortypes
+
+// Defines numeric codes for well-known errors in this package.
+const (
+	NoErrorCode = iota
+	TimeoutCode
+	BadInputCode
+	BlacklistedAppCode
+	BadServerResponseCode
+	FailedToRequestBidsCode
+	BidderTemporarilyDisabledCode
+	BlacklistedAcctCode
+	AcctRequiredCode
+	InvalidPrivacyConsentCode
+	UnknownErrorCode = 999
```

Comment:
Should we also have an `UnknownWarningCode`? They're in their own collections so the same unknown number being used makes a bit of sense to me.

Reply:
Makes sense, especially if we go with the gap idea

Reply:
I thought this would make sense, but it got messy when I tried to code it because `ReadErrorCode` is hardcoded to `UnknownErrorCode`. I would need to also add a detection of severity to that method to distinguish between known errors and warnings.. and at that point, do you think it's worth it? We don't call the function very often so perhaps the impact isn't bad. Thoughts?